tests/integration/test_record.py

```diff
@@ -347,3 +399,27 @@ def test_records_can_be_merged(app, records_not_merged_in_marcxml):
     with app.test_client() as client:
         assert client.get('/api/literature/111').status_code == 200
         assert client.get('/api/literature/222').status_code == 301
+
+
+def test_records_propagate_links_when_merged(app, records_to_be_merged):
+    before_record_update.disconnect(check_if_record_is_going_to_be_deleted)
```

Comment:
I agree. Rather than changing metadata and relying on the knowledge that there is some receiver somewhere doing the magic, I would rather see and explicit call to `.merge()`